src/pattern.js

```diff
@@ -314,6 +314,7 @@ CronPattern.prototype.handleStepping = function (conf, type) {
  */
 CronPattern.prototype.replaceAlphaDays = function (conf) {
 	return conf
+		.replace(/-sun/gi, "-7")
```

Comment:
A code-comment on this would be nice, with a quick explanation for future contributors